src/TuringTutorials.jl

```diff
@@ -9,12 +9,16 @@ using Plots
 using Requires
 using Weave
 
-export build_folder, build_all, verify_logs, tutorials
+const repo_dir = string(pkgdir(TuringTutorials))::String
```

Comment:
This should probably be `REPO_DIR` instead of `repo_dir` to clarify that it's a constant.